{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/oz/0.8.0/access/Ownable.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:118:1:\n    |\n118 | import \"../oz/0.8.0/access/Ownable.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/oz/0.8.0/access/Ownable.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:223:1:\n    |\n223 | import \"../oz/0.8.0/access/Ownable.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/d2/Signature_Verifier.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:224:1:\n    |\n224 | import \"./Signature_Verifier.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/d2/ENS_Registry.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:225:1:\n    |\n225 | import \"./ENS_Registry.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/d2/ENS_Resolver.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:226:1:\n    |\n226 | import \"./ENS_Resolver.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/utils/Context.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/SmarthER/mainnet/d2/d269323A7FD9a79aDf3Cb4f6fBDc49c6A1F9f460_Zapper_Mail_V1.sol:493:1:\n    |\n493 | import \"../utils/Context.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
